scripts/src/stellar.ts

```diff
@@ -202,6 +204,12 @@ export class Operations {
 		return (await this.server.operations().forTransaction(hash).call()).records[0] as PaymentOperationRecord;
 	}
 
+	@retry({ errorMessagePrefix: "failed to fetch burn operations record" })
+	public async getBurnRecords(hash: string): Promise<Array<ChangeTrustOperationRecord | SetOptionsOperationRecord>> {
+		const records = (await this.server.operations().forTransaction(hash).call()).records;
+		return [records[0] as ChangeTrustOperationRecord, records[1] as SetOptionsOperationRecord];
```

Comment:
This doesn't look like it returns burn records only. It seems like it would return any transaction with two operations.
Maybe add a method doc to explain what this `hash` needs to be.